_includes/site/layout/base/footer/links.html

```diff
@@ -11,11 +11,14 @@
   <ul class="links">
     <li><a href="{% link about/index.md %}">About</a></li>
     <li><a href="{% link faq/index.md %}">FAQ</a></li>
+    <li><a href="https://www.google.com/policies/terms/">Website Terms</a></li>
```

Comment:
Does Website refer to forsetisecurity.org?

Reply:
Yup, suggestions on different wording if that isn't preferred?

Reply:
I think it sounds okay as is.

Reply:
Done